tests/MongoDB.Driver.Tests/Search/SearchDefinitionBuilderTests.cs

```diff
@@ -261,6 +261,16 @@ public void EmbeddedDocument_typed()
                 subjectFamily.EmbeddedDocument(p => p.Children, subjectPerson.QueryString(p => p.LastName, "berg")),
                 "{ embeddedDocument: { path : 'Children', operator : { 'queryString' : { defaultPath : 'Children.ln', query : 'berg' } } } }");
         }
+        
+        [Fact]
+        public void Equals_with_array_should_render_supported_type()
+        {
+            var subjectTyped = CreateSubject<Person>();
+            
+            AssertRendered(
+                subjectTyped.Equals(p => p.Hobbies, "soccer"),
+                $"{{ equals: {{ path: 'hobbies', value: \"soccer\" }} }}");
```

Comment:
```suggestion
                """{{ equals: {{ path: 'hobbies', value: "soccer" }} }}""");
```
Small suggestion here. You don't need the `$` for interpolation, and you could use the raw string literal to avoid the need to escape the quotes

Reply:
Thanks, I think I can just remove the dollar sign and double brackets and use single quotes.